client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -421,14 +422,23 @@ const PatternAssembler = ( {
 			return;
 		}
 
-		handleContinue();
+		if ( isNewSite ) {
```

Comment:
Looks like it might be better to move this into the `onSubmit` function as it fires the `CONTINUE_TO_EDITOR_CLICK` event.

Reply:
I'm not sure what you mean. Can you be more clear about what logic and where to move it?

Reply:
Sorry, I mean the `trackEventContinue` function 😅

Reply:
oh, right, I missed that, thanks